package.json

```diff
@@ -16,7 +16,7 @@
     "test:debug": "node --inspect-brk ./node_modules/.bin/jest",
     "test:admin-ui": "jest tests/admin-ui-tests",
     "build": "preconstruct build",
-    "prepare": "preconstruct dev"
```

Comment:
I am personally against using `husky` or anything that hooks into my git workflows